keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/annealing/ResourcePersister.kt

```diff
@@ -0,0 +1,43 @@
+package com.netflix.spinnaker.keel.annealing
+
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.events.ResourceEvent
+import com.netflix.spinnaker.keel.events.ResourceEventType.CREATE
+import com.netflix.spinnaker.keel.events.ResourceEventType.DELETE
+import com.netflix.spinnaker.keel.events.ResourceEventType.UPDATE
+import com.netflix.spinnaker.keel.persistence.ResourceRepository
+import com.netflix.spinnaker.keel.plugin.ResourceHandler
+import com.netflix.spinnaker.keel.plugin.supporting
+import org.slf4j.LoggerFactory
+import org.springframework.context.ApplicationEventPublisher
+import org.springframework.stereotype.Component
+
+@Component
+class ResourcePersister(
```

Comment:
I just moved this class from `keel-api` to `keel-plugin` so it sits with the others involved in this workflow. Seemed more logical there.